{
  "uid" : "4fb3884439158e15",
  "name" : "Verify presence, visibility and attributes of the image in the section 1",
  "fullName" : "tests.start_unauthorized_page_test.TestStartUnauthorizedPageImage#test_su_04_01_verify_image_in_section1",
  "historyId" : "1aafb9d66326b6a4ae885f268ca195f1",
  "time" : {
    "start" : 1728697344547,
    "stop" : 1728697344547,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55dc847ab02a <unknown>\n#1 0x55dc844915e0 <unknown>\n#2 0x55dc844897eb <unknown>\n#3 0x55dc84479a09 <unknown>\n#4 0x55dc8447b744 <unknown>\n#5 0x55dc84479cbd <unknown>\n#6 0x55dc8447952f <unknown>\n#7 0x55dc84479422 <unknown>\n#8 0x55dc84477493 <unknown>\n#9 0x55dc84477ada <unknown>\n#10 0x55dc84493f41 <unknown>\n#11 0x55dc845258c5 <unknown>\n#12 0x55dc84505b22 <unknown>\n#13 0x55dc84524d7d <unknown>\n#14 0x55dc845058c3 <unknown>\n#15 0x55dc844d36b3 <unknown>\n#16 0x55dc844d468e <unknown>\n#17 0x55dc84775a2b <unknown>\n#18 0x55dc847799b1 <unknown>\n#19 0x55dc84762225 <unknown>\n#20 0x55dc8477a532 <unknown>\n#21 0x55dc8474738f <unknown>\n#22 0x55dc84799f28 <unknown>\n#23 0x55dc8479a0f3 <unknown>\n#24 0x55dc847a9e7c <unknown>\n#25 0x7fb093094ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"77d913adcb88dadc86d8a9c66857592b\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f4df4ede540>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55dc8479a0f3 \\\\u003Cunknown>\\\\n#24 0x55dc847a9e7c \\\\u003Cunknown>\\\\n#25 0x7fb093094ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55dc847ab02a <unknown>\nE       #1 0x55dc844915e0 <unknown>\nE       #2 0x55dc844897eb <unknown>\nE       #3 0x55dc84479a09 <unknown>\nE       #4 0x55dc8447b744 <unknown>\nE       #5 0x55dc84479cbd <unknown>\nE       #6 0x55dc8447952f <unknown>\nE       #7 0x55dc84479422 <unknown>\nE       #8 0x55dc84477493 <unknown>\nE       #9 0x55dc84477ada <unknown>\nE       #10 0x55dc84493f41 <unknown>\nE       #11 0x55dc845258c5 <unknown>\nE       #12 0x55dc84505b22 <unknown>\nE       #13 0x55dc84524d7d <unknown>\nE       #14 0x55dc845058c3 <unknown>\nE       #15 0x55dc844d36b3 <unknown>\nE       #16 0x55dc844d468e <unknown>\nE       #17 0x55dc84775a2b <unknown>\nE       #18 0x55dc847799b1 <unknown>\nE       #19 0x55dc84762225 <unknown>\nE       #20 0x55dc8477a532 <unknown>\nE       #21 0x55dc8474738f <unknown>\nE       #22 0x55dc84799f28 <unknown>\nE       #23 0x55dc8479a0f3 <unknown>\nE       #24 0x55dc847a9e7c <unknown>\nE       #25 0x7fb093094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728697344547,
      "stop" : 1728697345330,
      "duration" : 783
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728697345330,
      "stop" : 1728697345755,
      "duration" : 425
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55dc847ab02a <unknown>\n#1 0x55dc844915e0 <unknown>\n#2 0x55dc844897eb <unknown>\n#3 0x55dc84479a09 <unknown>\n#4 0x55dc8447b744 <unknown>\n#5 0x55dc84479cbd <unknown>\n#6 0x55dc8447952f <unknown>\n#7 0x55dc84479422 <unknown>\n#8 0x55dc84477493 <unknown>\n#9 0x55dc84477ada <unknown>\n#10 0x55dc84493f41 <unknown>\n#11 0x55dc845258c5 <unknown>\n#12 0x55dc84505b22 <unknown>\n#13 0x55dc84524d7d <unknown>\n#14 0x55dc845058c3 <unknown>\n#15 0x55dc844d36b3 <unknown>\n#16 0x55dc844d468e <unknown>\n#17 0x55dc84775a2b <unknown>\n#18 0x55dc847799b1 <unknown>\n#19 0x55dc84762225 <unknown>\n#20 0x55dc8477a532 <unknown>\n#21 0x55dc8474738f <unknown>\n#22 0x55dc84799f28 <unknown>\n#23 0x55dc8479a0f3 <unknown>\n#24 0x55dc847a9e7c <unknown>\n#25 0x7fb093094ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728697345330,
        "stop" : 1728697345754,
        "duration" : 424
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55dc847ab02a <unknown>\n#1 0x55dc844915e0 <unknown>\n#2 0x55dc844897eb <unknown>\n#3 0x55dc84479a09 <unknown>\n#4 0x55dc8447b744 <unknown>\n#5 0x55dc84479cbd <unknown>\n#6 0x55dc8447952f <unknown>\n#7 0x55dc84479422 <unknown>\n#8 0x55dc84477493 <unknown>\n#9 0x55dc84477ada <unknown>\n#10 0x55dc84493f41 <unknown>\n#11 0x55dc845258c5 <unknown>\n#12 0x55dc84505b22 <unknown>\n#13 0x55dc84524d7d <unknown>\n#14 0x55dc845058c3 <unknown>\n#15 0x55dc844d36b3 <unknown>\n#16 0x55dc844d468e <unknown>\n#17 0x55dc84775a2b <unknown>\n#18 0x55dc847799b1 <unknown>\n#19 0x55dc84762225 <unknown>\n#20 0x55dc8477a532 <unknown>\n#21 0x55dc8474738f <unknown>\n#22 0x55dc84799f28 <unknown>\n#23 0x55dc8479a0f3 <unknown>\n#24 0x55dc847a9e7c <unknown>\n#25 0x7fb093094ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"77d913adcb88dadc86d8a9c66857592b\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728697167335,
      "stop" : 1728697167335,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728697345787,
      "stop" : 1728697345853,
      "duration" : 66
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Start Unauthorized Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "start_unauthorized_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestStartUnauthorizedPage"
  }, {
    "name" : "host",
    "value" : "fv-az570-490"
  }, {
    "name" : "thread",
    "value" : "2304-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.start_unauthorized_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 28,
        "unknown" : 0,
        "total" : 30
      },
      "items" : [ {
        "uid" : "828bc5c7af8c402",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/828bc5c7af8c402",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x556714ee902a <unknown>\n#1 0x556714bcf5e0 <unknown>\n#2 0x556714bc77eb <unknown>\n#3 0x556714bb7a09 <unknown>\n#4 0x556714bb9744 <unknown>\n#5 0x556714bb7cbd <unknown>\n#6 0x556714bb752f <unknown>\n#7 0x556714bb7422 <unknown>\n#8 0x556714bb5493 <unknown>\n#9 0x556714bb5ada <unknown>\n#10 0x556714bd1f41 <unknown>\n#11 0x556714c638c5 <unknown>\n#12 0x556714c43b22 <unknown>\n#13 0x556714c62d7d <unknown>\n#14 0x556714c438c3 <unknown>\n#15 0x556714c116b3 <unknown>\n#16 0x556714c1268e <unknown>\n#17 0x556714eb3a2b <unknown>\n#18 0x556714eb79b1 <unknown>\n#19 0x556714ea0225 <unknown>\n#20 0x556714eb8532 <unknown>\n#21 0x556714e8538f <unknown>\n#22 0x556714ed7f28 <unknown>\n#23 0x556714ed80f3 <unknown>\n#24 0x556714ee7e7c <unknown>\n#25 0x7fa49cc94ac3 <unknown>",
        "time" : {
          "start" : 1728611035748,
          "stop" : 1728611035748,
          "duration" : 0
        }
      }, {
        "uid" : "92c2e1cf616ea6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/92c2e1cf616ea6",
        "status" : "passed",
        "time" : {
          "start" : 1728525075759,
          "stop" : 1728525075862,
          "duration" : 103
        }
      }, {
        "uid" : "54f51bb498fd98eb",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/54f51bb498fd98eb",
        "status" : "passed",
        "time" : {
          "start" : 1728438746865,
          "stop" : 1728438746968,
          "duration" : 103
        }
      }, {
        "uid" : "55a87ba1a77950c5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/55a87ba1a77950c5",
        "status" : "passed",
        "time" : {
          "start" : 1728352339480,
          "stop" : 1728352339562,
          "duration" : 82
        }
      }, {
        "uid" : "b845bcea7062b5d8",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/b845bcea7062b5d8",
        "status" : "passed",
        "time" : {
          "start" : 1728266942558,
          "stop" : 1728266942653,
          "duration" : 95
        }
      }, {
        "uid" : "cd7c6abb09e4b70f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/cd7c6abb09e4b70f",
        "status" : "passed",
        "time" : {
          "start" : 1728179881042,
          "stop" : 1728179881126,
          "duration" : 84
        }
      }, {
        "uid" : "f46ffcc9b8cad81c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/f46ffcc9b8cad81c",
        "status" : "passed",
        "time" : {
          "start" : 1728093137954,
          "stop" : 1728093138044,
          "duration" : 90
        }
      }, {
        "uid" : "7c39f8911a805d0c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/7c39f8911a805d0c",
        "status" : "passed",
        "time" : {
          "start" : 1728006841054,
          "stop" : 1728006841134,
          "duration" : 80
        }
      }, {
        "uid" : "4a6b1c683c33db30",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/4a6b1c683c33db30",
        "status" : "passed",
        "time" : {
          "start" : 1727920504852,
          "stop" : 1727920504940,
          "duration" : 88
        }
      }, {
        "uid" : "9f65cd4770a0e613",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/9f65cd4770a0e613",
        "status" : "passed",
        "time" : {
          "start" : 1727833911337,
          "stop" : 1727833911428,
          "duration" : 91
        }
      }, {
        "uid" : "caced04b86460d78",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/caced04b86460d78",
        "status" : "passed",
        "time" : {
          "start" : 1727748037548,
          "stop" : 1727748037647,
          "duration" : 99
        }
      }, {
        "uid" : "4eea73c9ce32a6b9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/4eea73c9ce32a6b9",
        "status" : "passed",
        "time" : {
          "start" : 1727661317878,
          "stop" : 1727661317974,
          "duration" : 96
        }
      }, {
        "uid" : "5a802f1a603eaea9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/5a802f1a603eaea9",
        "status" : "passed",
        "time" : {
          "start" : 1727575114724,
          "stop" : 1727575114825,
          "duration" : 101
        }
      }, {
        "uid" : "217ce5b24cabc7e9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/217ce5b24cabc7e9",
        "status" : "passed",
        "time" : {
          "start" : 1727488377102,
          "stop" : 1727488377184,
          "duration" : 82
        }
      }, {
        "uid" : "3da8f5bbb646c140",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/3da8f5bbb646c140",
        "status" : "passed",
        "time" : {
          "start" : 1727402112818,
          "stop" : 1727402112901,
          "duration" : 83
        }
      }, {
        "uid" : "296bdb1c1e2e8217",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/296bdb1c1e2e8217",
        "status" : "passed",
        "time" : {
          "start" : 1727315621729,
          "stop" : 1727315621810,
          "duration" : 81
        }
      }, {
        "uid" : "c7684ed8b7472eeb",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/c7684ed8b7472eeb",
        "status" : "passed",
        "time" : {
          "start" : 1727229217532,
          "stop" : 1727229217617,
          "duration" : 85
        }
      }, {
        "uid" : "8deecd5ba01aa72e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/8deecd5ba01aa72e",
        "status" : "passed",
        "time" : {
          "start" : 1727142745692,
          "stop" : 1727142745797,
          "duration" : 105
        }
      }, {
        "uid" : "aadf9c04a0d083ec",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/aadf9c04a0d083ec",
        "status" : "passed",
        "time" : {
          "start" : 1727056260418,
          "stop" : 1727056260494,
          "duration" : 76
        }
      }, {
        "uid" : "efa249dbe38a39c9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/40//#testresult/efa249dbe38a39c9",
        "status" : "passed",
        "time" : {
          "start" : 1726970190091,
          "stop" : 1726970190169,
          "duration" : 78
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "4fb3884439158e15.json",
  "parameterValues" : [ ]
}